int32 where
bool red_zone
int32 route
bool ToShootSpot
float32 speed_multiplier
float32 k
